site/src/pages/TemplatePage/TemplateInsightsPage/utils.ts

```diff
@@ -1,15 +1,22 @@
-import { addDays, addHours, format, startOfDay, startOfHour } from "date-fns";
+import {
+  addDays,
+  addHours,
+  format,
+  startOfDay,
+  startOfHour,
+  isToday as isTodayDefault,
+} from "date-fns";
 
 export function getDateRangeFilter({
   startDate,
   endDate,
-  now,
-  isToday,
+  now = new Date(),
+  isToday = isTodayDefault,
 }: {
   startDate: Date;
   endDate: Date;
-  now: Date;
-  isToday: (date: Date) => boolean;
+  now?: Date;
+  isToday?: (date: Date) => boolean;
```

Comment:
can we break apart the type signature and the destructing? just gets so messy this way